tests.py

```diff
@@ -299,7 +302,7 @@ def test_solve_and_derivative(self):
         A, b, c, cone_dims = utils.least_squares_eq_scs_data(m, n)
```

Comment:
Should `test_solve_and_derivative`, `test_warmstart`, and `test_threading` be refactored into a new class, `TestSCS`?

Reply:
Fixed.